docs/operator.md

```diff
@@ -242,6 +242,75 @@ Add the annotation to the AgentServiceConfig:
 oc annotate --overwrite AgentServiceConfig agent unsupported.agent-install.openshift.io/assisted-image-service-skip-verify-tls=true
 ```
 
+### Custom headers and query parameters for Assisted Image Service OS Image Download
+
+It is possible to specify custom headers and query parameters to be used when downloading OS images.
+
+A config map to contain the headers and query parameters to be used should be created.
+The easiest approach is to create up to two files as required, one file to hold any headers, the other to hold query parameters.
+Use these files to create a secret. The files may be discarded after creation of the secret.
+
+```
+       cat <<EOF >> headers
+         {
+         	"header1": "header1value",
+         	"header2": "header2value",
+         }
+       EOF
+       cat <<EOF >> query_params
+         {
+         	"param1": "value1",
+         	"param2": "value2",
+         }
+       EOF
+       
+       oc create secret generic -n multicluster-engine os-images-http-auth --from-file=./query_params --from-file=./headers
+```
+
+Then in the `AgentServiceConfig`, this Secret should be referenced, in `OSImageAdditionalParamsRef` 
+The headers and query params contained within the ConfigMap referenced by `OSImageAdditionalParamsRef` will then be used when pulling osImages.
```

Comment:
Same here, should it be a ConfigMap or a Secret?

Reply:
agreed. It's a Secret and not a ConfigMap, I have updated both locations to reflect